4_Marketing-Support/support.md

```diff
@@ -1,8 +1,8 @@
 # Handling Support Requests
 
-Email support is managed by [Intercom](https://app.intercom.io), a customer messaging platform that supports multiple channels. Support requests are sent via our contact forms, directly to support email address, social media pages (Facebook and Twitter), and through Intercom's Messenger (live chat).
+Email support is managed by [Help Scout](https://www.helpscout.com/), a customer messaging platform that supports multiple channels. Support requests are sent via our contact forms, directly to support email address, in-app webhooks, social media pages (Facebook and Twitter), and through Help Scout Beacons (live chat).
```

Comment:
No social media pages afaik?

Reply:
Right now no. I'll remove that.

Help Scout has confirmed they are working on that now (they interviewed me how we use social media for support).